HiP-DataStore/Core/ReadModel/CacheDatabaseManager.cs

```diff
@@ -93,7 +93,10 @@ private void ApplyEvent(IEvent ev)
                     break;
 
                 case ExhibitDeleted e:
-                    _db.GetCollection<Exhibit>(ResourceType.Exhibit.Name).DeleteOne(x => x.Id == e.Id);
+                    var deleteExhibit = _db.GetCollection<Exhibit>(ResourceType.Exhibit.Name).AsQueryable().First(x => x.Id == e.Id);
```

Comment:
The code for `*Deleted`-events is quite repetitive, maybe we can extract that into a method like:
```C#
private void MarkDeleted<T>(ResourceType resourceType, int entityId) where T : ContentBase
{
    var collection = _db.GetCollection<T>(resourceType.Name);
    var entity = collection.AsQueryable().First(x => x.Id == entityId);
    entity.Status = ContentStatus.Deleted;
    collection.ReplaceOne(x => x.Id == entityId, entity);           
}
```